web/src/components/Form.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div>
+    <form @submit.prevent="onSubmit">
+      <div class="form-row">
+        <div class="form-group input">
+
+          <label for="start-date">Contract start date</label>
+          <input
+            type="date"
+            class="form-control"
+            id="start-date"
+            v-model="startDate"
+            @blur="$v.startDate.$touch()">
```

Comment:
No need for the brackets in a event 
```suggestion
            @blur="$v.startDate.$touch">
```